scripts/sanitycheck

```diff
@@ -2646,7 +2646,8 @@ class TestSuite:
                 cr = csv.DictReader(fp)
                 instance_list = []
                 for row in cr:
-                    if row["status"] in filter_status:
+                    # FIXME: ignore header row.
```

Comment:
Can we explain the `row['arch'] == 'arch'` in the comment in a little more detail.

Something like:

```
# We might have header rows intermixed, detect them by the the 'arch' field being set to 'arch'
```